allennlp/interpret/attackers/input_reduction.py

```diff
@@ -86,7 +113,8 @@ def attack_from_json(self, inputs: JsonDict = None,
 def _remove_one_token(instance: Instance,
                       input_field_to_attack: str,
                       grads: np.ndarray,
-                      ignore_tokens: List[str]) -> Tuple[Instance, int]:
+                      ignore_tokens: List[str],
+                      beam_size: int = 1) -> List[Tuple[Instance, int]]:
```

Comment:
This is a private method.  You don't need the default value here (and it just gives another place for defaults to mismatch, as they currently do).